### DISCUSSION TITLE
Which member of the Simpsons family is a vegetarian?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#184] [Galaxy Trivia] [answered]
**Hint:** ☀

**Comment by thenithinbalaji** [comment, marked as answer]
Lisa Simpson is a vegetarian in the Simpsons family. She decided to become a vegetarian in the episode "Lisa the Vegetarian" which aired during the show's seventh season. In the episode, Lisa learns about the ethical treatment of animals and decides to stop eating meat, much to the chagrin of her meat-loving father, Homer.
